ghost/members-api/lib/MembersAPI.js

```diff
@@ -218,6 +221,12 @@ module.exports = function MembersAPI({
             return member;
         }
         const newMember = await users.create({name, email, labels, newsletters, attribution});
+
+        // Notify staff users of new free member signup
+        if (labsService.isSet('emailAlerts')) {
+            await staffService.notifyFreeMemberSignup(newMember.toJSON());
```

Comment:
I'm not sure why this is done here, rather than in the create method for member repo. I get that we don't want to send the free signup email when creating paid members, but a flag could be passed when creating a member by Stripe? 

We shouldn't be having side effects like this happening in a method with a name implying a read operation IMO

Reply:
>  get that we don't want to send the free signup email when creating paid members, but a flag could be passed when creating a member by Stripe?

Yea agree adding a flag to `create` method from here and triggering it in there definitely makes more sense, I can clean this up today. Only reason it went in there was this was the first alert I added and this was the quickest way.

Reply:
Okay cool - thanks :)